nestadia-wgpu/src/main.rs

```diff
@@ -30,6 +35,24 @@ struct Opt {
     start_paused: bool,
 }
 
+struct Logger;
+
+impl log::Log for Logger {
+    fn enabled(&self, metadata: &log::Metadata) -> bool {
+        metadata.level() <= log::Level::Info
+    }
+
+    fn log(&self, record: &log::Record) {
+        if self.enabled(record.metadata()) {
+            println!("{} - {}", record.level(), record.args());
+        }
+    }
+
+    fn flush(&self) {}
+}
+
+static LOGGER: Logger = Logger;
+
```

Comment:
Ahh ok, I hadn't noticed it (didn't think to look at the server cargo.toml). I will update that. I assume for now I should do the same for libretro (but taking it from the env instead of an arg) until we can properly hook on the libretro logging system?